assets/js/DateConversion.js

```diff
@@ -0,0 +1,42 @@
+export default function (date, precision) {
+  let [day, month, year] = date.split("/");
+  day = Number(day);
+  month = Number(month);
+  year = Number(year);
+  let currentGregorianDate = new Date(year, month - 1, day);
+
+  const PRECISION_DAY_MONTH_YEAR = "11";
```

Comment:
It doesn't look like this constant is ever used so we can remove it from the code. Thanks Ruby

Reply:
I've removed as only a minor edit. Resolving issue